woocommerce-services.php

```diff
@@ -158,12 +158,8 @@ static function load_tracks_for_activation_hooks() {
 			return new WC_Connect_Tracks( $logger );
 		}
 
-		static function plugin_activation() {
-			$tracks = self::load_tracks_for_activation_hooks();
-			$tracks->opted_in();
-		}
-
 		static function plugin_deactivation() {
+			update_option( 'wc_connect_tos_accepted', false );
```

Comment:
Let's remove this change in favor of handling it in https://github.com/Automattic/woocommerce-services/issues/776.